docs/DeveloperGuide.md

```diff
@@ -831,30 +836,66 @@ The project demonstrated a high level of effort and collaboration, resulting in
 
 The following planned enhancements address known feature flaws identified during the PE-D phase. Each enhancement specifically describes the feature flaw and the proposed solution, providing details on how the feature will be improved. This section lists 3 planned enhancements, adhering to the team size x 2 limit.
```

Comment:
We are only allowed to have 10 planned enhancements, so need to remove the 2 that are least relevant